cmd/controller/main.go

```diff
@@ -234,19 +239,21 @@ func main() {
 		rs = append(rs, gasController)
 	}
 
-	kubeInformerFactory.Start(ctx.Done())
-	agonesInformerFactory.Start(ctx.Done())
+	runLeaderElection(ctx, cancel, logger, ctlConf, kubeClient, func(ctx context.Context) {
```

Comment:
now that this is basically "run this function when I'm leader", maybe `whenLeader`. Then it's more obvious what the function is.

Reply:
Done!